android/src/main/java/com/github/yamill/orientation/ThrottleUtils.kt

```diff
@@ -0,0 +1,26 @@
+package com.github.yamill.orientation
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+
+fun <T, R> throttleLatest(
```

Comment:
I got this from https://stackoverflow.com/questions/50858684/kotlin-android-debounce

Reply:
couldn't hurt to add this as a comment

Reply:
updated!